docs/docsite/rst/playbook_guide/playbooks_module_defaults.rst

```diff
@@ -102,57 +102,10 @@ Setting a default AWS region for specific EC2-related modules.
-
-+---------+---------------------------+-----------------+
-| Group   | Purpose                   | Ansible Version |
-+=========+===========================+=================+
-| aws     | Amazon Web Services       | 2.7             |
-+---------+---------------------------+-----------------+
-| azure   | Azure                     | 2.7             |
-+---------+---------------------------+-----------------+
-| gcp     | Google Cloud Platform     | 2.7             |
-+---------+---------------------------+-----------------+
-| k8s     | Kubernetes                | 2.8             |
-+---------+---------------------------+-----------------+
-| os      | OpenStack                 | 2.8             |
-+---------+---------------------------+-----------------+
-| acme    | ACME                      | 2.10            |
-+---------+---------------------------+-----------------+
-| docker* | Docker                    | 2.10            |
-+---------+---------------------------+-----------------+
-| ovirt   | oVirt                     | 2.10            |
-+---------+---------------------------+-----------------+
-| vmware  | VMware                    | 2.10            |
-+---------+---------------------------+-----------------+
-
-* The `docker_stack <docker_stack_module>`_ module is not included in the ``docker`` defaults group.
+Module default groups allow to provide common parameters to groups of modules that belong together. Collections can define such groups in their ``meta/runtime.yml`` file.
 
-Use the groups with ``module_defaults`` by prefixing the group name with ``group/`` - for example ``group/aws``.
-
-In a playbook, you can set module defaults for whole groups of modules, such as setting a common AWS region.
-
-.. code-block:: YAML
-
-    # example_play.yml
-    - hosts: localhost
-      module_defaults:
-        group/aws:
-          region: us-west-2
-      tasks:
-      - name: Get info
-        aws_s3_bucket_info:
-
-      # now the region is shared between both info modules
-
-      - name: Get info
-        ec2_ami_info:
-          filters:
-            name: 'RHEL*7.5*'
-
-In ansible-core 2.12, collections can define their own groups in the ``meta/runtime.yml`` file. ``module_defaults`` does not take the ``collections`` keyword into account, so the fully qualified group name must be used for new groups in ``module_defaults``.
+.. note::
+    ``module_defaults`` does not take the ``collections`` keyword into account, so the fully qualified group name must be used for new groups in ``module_defaults``.
 
 Here is an example ``runtime.yml`` file for the ``ns.coll`` collection.
 This file defines an action group named ``ns.coll.my_group`` and places the ``sample_module`` from ``ns.coll`` and ``another_module`` from ``another.collection`` into the group.
```

Comment:
Can you add the following (or something similar) after the code block below (I cannot add a comment there directly):
```rst
More information on meta/runtime.yml, including the complete format for `action_groups`, can be found in :ref:`meta_runtime_yml`.
```

Reply:
```suggestion
This file defines an action group named ``ns.coll.my_group`` and places the ``sample_module`` from ``ns.coll`` and ``another_module`` from ``another.collection`` into the group.
```

@felixfontein [Like this](https://github.com/ansible/ansible-documentation/pull/1893/commits/5a021e90570ec2fe2020a8b5cbf9f0320dc9b8e4)?